### PR TITLE
PIMS-1426, PIMS-1355

### PR DESCRIPTION
**Pull request by jdtoombs**
Updated unit tests to include ProjectNumber where appropriate

### CODE REVIEW

File: frontend/src/components/maps/leaflet/Map.tsx [hunk 1]
```diff
@@ -108,12 +110,21 @@ const Map: React.FC<MapProps> = ({
 
   const handleViewportChange = () => {
     const bounds = getBounds();
-    const { address, municipality, agencies, classificationId, minLotSize, maxLotSize } = mapFilter;
+    const {
```
Comment (goemen) [inline frontend/src/components/maps/leaflet/Map.tsx:113]: Just nitpicking, remove whitespace

File: frontend/src/pages/MapView.tsx [hunk 2]
```diff
@@ -63,8 +64,15 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
     if (!e || !e.bounds) {
       return null;
     }
-    const { address, municipality, agencies, classificationId, minLotSize, maxLotSize } =
-      e.filter ?? {};
+    const {
```
Comment (goemen) [inline frontend/src/pages/MapView.tsx:67]: whitespace

File: frontend/src/components/maps/leaflet/Map.tsx (hunk 1 above)
Comment (jdtoombs) [reply frontend/src/components/maps/leaflet/Map.tsx:113]: On it!

Comment (jdtoombs) [reply frontend/src/components/maps/leaflet/Map.tsx:113]: Looks like this is auto-formatted. Going to dismiss for now

**Pull request closed without merging**